WooCommerce/UITestsFoundation/Screens/Orders/UnifiedOrderScreen.swift

```diff
@@ -59,6 +59,10 @@ public final class UnifiedOrderScreen: ScreenObject {
     ///
     private var addProductButton: XCUIElement { addProductButtonGetter(app) }
 
+    /// Add Custom Amount button in Product section.
+    ///
```

Comment:
i'm aware that the comments are mostly existing ones and you're most likely adding it to match what's on the screen, but do you feel these comments are necessary? i think the naming is clear enough that we could do without the comments. wdyt?

Reply:
Yes, I did this because of the consistency reasons. It didn't feel right not to add it, but it also doesn't feel right to remove it from all other elements in this PR, because of the noise it will create.